tests/drivers/dma/loop_transfer/CMakeLists.txt

```diff
@@ -7,4 +7,4 @@ project(loop_transfer)
 FILE(GLOB app_sources src/*.c)
 target_sources(app PRIVATE ${app_sources})
 
-zephyr_code_relocate(FILES src/test_dma_loop.c LOCATION ${CONFIG_DMA_LOOP_TRANSFER_RELOCATE_SECTION}_RODATA_BSS)
+zephyr_code_relocate(FILES src/tst_dma:_loop.c LOCATION ${CONFIG_DMA_LOOP_TRANSFER_RELOCATE_SECTION}_RODATA_BSS)
```

Comment:
Was this meant to be src/tst_dma_loop.c or was the colon intentional?

Reply:
huh, i made ONE mistake in the original pr, trying to fix it introduced other issues every time, sometimes it is better to just start over...

Reply:
Isn't that how it always works?